docker/services/contrail/contrail-config-database.yaml

```diff
@@ -68,6 +68,30 @@ outputs:
       service_config_settings: {get_attr: [ContrailBase, role_data, service_config_settings]}
       docker_config:
         step_4:
+         contrail_config_database_nodemgr:
+            image: {get_attr: [ContrailBase, role_data, contrail_nodemgr_image_name]}
+            environment:
+              list_concat:
+                - if:
+                  - contrail_configdb_min_disk_gb_unset
+                  - []
+                  - - list_join:
+                      - ''
+                      - - 'CONFIG_DATABASE_NODEMGR__DEFAULTS__minimum_diskGB='
+                        - {get_param: ContrailConfigDBMinDiskGB}
```

Comment:
it is needed to add the ContrailConfigDBMinDiskGB as a parameter in this file (see parameters section above)